3001-4000/LIT3999Ofetun.xml

```diff
@@ -1,11 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
 schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT3999Ofetun">
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT3999Ofetun">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
             <title xml:lang="gez" xml:id="t1">ኦፍጡነ፡ ረድኤት፡</title>
             <title xml:lang="gez" corresp="#t1" type="normalized">ʾO-fǝṭuna radʾet</title>
+            <title xml:lang="en" corresp="#t1">Oh you, quick of help</title>
```

Comment:
Oh you, quick of help...
or
Oh you, quick in help...?

Reply:
I don't know. And this is actually the record I wanted to delete, https://github.com/BetaMasaheft/Documentation/issues/1312...